tests/test_core.py

```diff
@@ -131,6 +131,11 @@ def test_parse_bad(self, value) -> None:
         with pytest.raises(ValueError):
             Address.parse(value)
 
+    @pytest.mark.parametrize("value", ["1.2.3.4"])
+    def test_bad_host(self, value) -> None:
+        with pytest.raises(TypeError):
+            _ = Address(value)
```

Comment:
You don't need to assign it to anything. See the test above for example.
```suggestion
            Address(value)
```

Reply:
Man, and I was wondering about doing it the same way as `parse_bad`.

Reply:
Also, I think I'm going to change the 'bad value' passed in here (to `127.0.0.1`) to follow suit with the rest of the tests. No need to be unnecessarily different.